contracts/cw20-base/src/state.rs

```diff
@@ -23,6 +24,14 @@ pub struct MinterData {
     pub cap: Option<Uint128>,
 }
 
+#[derive(Serialize, Deserialize, Clone, PartialEq, JsonSchema, Debug)]
+pub struct MarketingInfo {
+    pub project: Option<String>,
+    pub description: Option<String>,
+    pub marketing: Option<Addr>,
+    pub logo: Option<Logo>,
```

Comment:
On reflection, I think TokenInfo is updated on minting and consulted a number of places.
It may make sense to store TokenInfo and MarketingInfo separately. And Logo in a 3rd section (which may well be empty, use `may_load` to check it)

Reply:
But pulling out Logo is the main issue